### PR TITLE
Mobifies transposed scientist and shades

### PR DESCRIPTION
**Pull request by CheffieGithub**
## About the PR 
changes obj critter crunched and shade into mob critters. 
they both have funky void speak.

## Why's this needed? 
obj critters bad

### CODE REVIEW

**Review by Tarmunora — APPROVED** [review]
will require some changes to secrets when merged

File: code/mob/living/critter/adventure.dm [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+		msgs.flush(SUPPRESS_LOGS)
+		user.lastattacked = target
+		ON_COOLDOWN(src, "limb_cooldown", 3 SECONDS)
+
+/*
+	Extralines I haven't used but can be later, here to preserve them.
+	orginally used for obj critter thinking messages
+
+	src.say(pick("Hey..you! Help! Help me please!","I need..a doctor...","Someone...new? Help me...please.","Are you real?"))
+	src.say(pick("Cut the power! It's about to go critical, cut the power!","I warned them. I warned them the system wasn't ready.","Shut it down!","It hurts, oh God, oh God."))
```
Comment (TobleroneSwordfish) [inline code/mob/living/critter/adventure.dm:107]: could put these behind a chance in `Life`?

File: code/mob/living/critter/adventure.dm [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+		add_hh_flesh(src.health_brute, src.health_brute_vuln)
+		add_hh_flesh_burn(src.health_burn, src.health_burn_vuln)
+
+	critter_attack(var/mob/target)
+		if (target.lying || is_incapacitated(target))
+			src.set_a_intent(INTENT_HELP)
+		else
+			src.set_a_intent(INTENT_HARM)
+		src.chase_lines(target)
+		src.hand_attack(target)
```
Comment (TobleroneSwordfish) [inline code/mob/living/critter/adventure.dm:55]: This should be `critter_basic_attack` now I think

File: code/mob/living/critter/adventure.dm [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+
+	src.speak(pick("namlugallu ha-lam ina lugal-šaà-lá-sù...","ù da-rí-sè šeš...","á-e-me-en ìri-zé-er igi-bad!","inim...kí ina ki-dul, ina e-ùr, ina ki-bad-rá, hé-àm-me-àm...", "ìri-kúr...díb, ìri...ar, e-zé...", "galam, gamar ganzer, gíbil píri! ul, ul! súkud..."))
+	// mankind destroyed the merciful king // sleep forever, brethren // i am one who lost my footing and opened my eyes // to seek or find the right words, the armor, the secret point, the distant places, that is our wish // to ascend, overwhelming darkness, burning bright! shine! shine! shine brightly!
+
+	src.speak(pick("ina urudu e-re-sì-ki-in kala libir arza ina SÚKUD ZAL.", "i.menden ina nam-ab-ba issa, nam-nu-tar  nam-diir, i.menden lúní-áa...","bar...gub ina bàd-šul-hi...","šidim ak ina libir išgal, diir ak ina agrun, ul-šár-ra, zà-mí!", "ùru pàd gíg, ina gidim niin!"))
+	// the copper servant mends the rights of the FLASH OF DAWN // we are the elder shades, ill-fated divinities, we are the temple servants...// step outside the outer wall
+	// architect of the ancient throne, god of the inner sanctuary, jubilation, praise! // watchfire reveals night, the darkened monstrosity
```
Comment (TobleroneSwordfish) [inline code/mob/living/critter/adventure.dm:201]: same here

**Review by TobleroneSwordfish — CHANGES_REQUESTED** [review]

**Comment by CheffieGithub** [comment]
life is a good call

**Pull request merged**